techmatters/hrm
# Review & Enhance Service Tests for Contact endpoints

## Description
Reworked `hrm-service/service-tests/contacts.test.js` tests to use (mostly) `jest-each` and added test cases (for query params, single search param, date filters, among others). Removed `CaseAudits` tests, instead checking against `Audits` table (not really necessary, maybe we want to remove this).

### Checklist
- [x] [Corresponding issue has been opened](https://bugs.benetech.org/browse/CHI-989)

### Verification steps
- Check that CI is still passing locally and remotely after the changes.
- Manual review to make sure test cases makes sense (:sweat_smile: )?
